pkg/controller/worker/machines.go

```diff
@@ -247,3 +260,12 @@ func createDiskSpec(volume v1alpha1.Volume, workerName string, machineImage stri
 
 	return disk, nil
 }
+
+// SanitizeGcpLabel will sanitize the label/value base on the gcp label Restrictions
+func SanitizeGcpLabel(label string) string {
```

Comment:
Thank you! Looks good, just a unit test for this function is missing. :)

Reply:
good suggestion. done